File: 74751_lab1_task4.py
# 74751_lab1_task4.py
import math

c0 = float(input("Enter initial drug concentration (C0): "))
k = float(input("Enter elimination rate constant (k): "))
t = float(input("Enter time (t): "))

c = c0 * math.exp(-k * t)
percentage = (c / c0) * 100

print(f"Current concentration (C): {c}")
print(f"Percentage of drug remaining: {percentage}%")

File: 74751_lab1_task4.cpp
// 74751_lab1_task4.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    double c0, k, t;
    cout << "Enter initial drug concentration (C0): ";
    cin >> c0;
    cout << "Enter elimination rate constant (k): ";
    cin >> k;
    cout << "Enter time (t): ";
    cin >> t;
    
    double c = c0 * exp(-k * t);
    double percentage = (c / c0) * 100;
    
    cout << "Current concentration (C): " << c << endl;
    cout << "Percentage of drug remaining: " << percentage << "%" << endl;
    
    return 0;
}
